web/ics/ics_creator.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Syndeo.IcsCreator do
+
+  def create_ics(date, description) do
+    events = [
+      %ICalendar.Event{
+        summary: "Wednesday night meal",
+        dtstart: start_time(date),
+        dtend: end_time(date),
+        description: "Come enjoy some #{description} with us.",
+        location: "3030 Virginia Ave, Collinsville, VA",
```

Comment:
not sure if it requires, but probably a good idea... Apple found the right address when i added the ics to Calendar.